src/windows/osd.c

```diff
@@ -505,3 +505,45 @@ void freeifaddrs(struct ifaddrs *ifa)
 		ifa = next;
 	}
 }
+
+ssize_t ofi_writev_socket(SOCKET fd, const struct iovec *iovec, size_t iov_cnt)
+{
+	ssize_t size;
+	int ret, i;
+	WSABUF *wsa_buf;
+	DWORD flags = 0;
+
+	wsa_buf = (WSABUF *)alloca(iov_cnt * sizeof(WSABUF));
+
+	for (i = 0; i < iov_cnt; i++) {
+		wsa_buf[i].buf = (char *)iovec[i].iov_base;
+		wsa_buf[i].len = iovec[i].iov_len;
+	}
```

Comment:
For the life of me, I can't figure out where struct iovec comes from for an application.  It's part of the libfabric API, but I don't think any windows file actually defines it.  We define it internally in include/windows/uio.h, which makes me think that it isn't defined for apps.  So... apps need to define this structure themselves prior to including fabric.h?  I need to look into this.